.github/workflows/ios-maestro.yml

```
@@ -100,46 +120,251 @@ jobs:
+                    echo "hash=$hash" >> "$GITHUB_OUTPUT"
+                    echo "Native fingerprint (${{ matrix.target }}): $hash"
+
+            # A cache miss leaves whatever this persistent self-hosted runner
+            # already had on disk from a previous fingerprint untouched —
+            # actions/checkout does not remove gitignored paths, and
+            # actions/cache/restore only ever adds/overwrites on a hit, never
+            # deletes on a miss. Clearing first guarantees the later `ditto`
+            # packaging step always overlays onto an empty directory instead
+            # of a stale Base.app that could still contain frameworks or
+            # resources this fingerprint's native change removed.
+            -   name: Clear native app cache directory
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)